scripts/kind-e2e/e2e.sh

```diff
@@ -145,8 +153,34 @@ function create_subm_vars() {
   subm_broker_ns=submariner-k8s-broker
 }
 
+function get_globalip() {
+    svcname=$1
+    gip=
+    attempt_counter=0
+    max_attempts=5
```

Comment:
@tpantelis gave a comment about 5 seconds on my PR, which I think is valid. Let's increase the max_attempts to 30 secs.

Reply:
you can also take a look at the other comments and make necessary changes - https://github.com/submariner-io/submariner/pull/363/files